mmv1/templates/terraform/custom_flatten/bigtable_app_profile_routing.tmpl

```diff
@@ -12,9 +12,16 @@
 */ -}}
 func flatten{{$.GetPrefix}}{{$.TitlelizeProperty}}(v interface{}, d *schema.ResourceData, config *transport_tpg.Config) interface{} {
 	if v == nil {
+		d.Set("row_affinity", nil)
```

Comment:
Can we always set it as nil, before this if, and only set to true if line 19 happens?